site/source/cookbook/widgets/pass-and-receive-props.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: docs
+category: widgets
+title: Pass and receive properties with widgets
+overview: Pass down properties to a widget and receive them from within a widget
+---
+
+## Objective
+
+This recipe demonstrates how to pass properties down to a widget and how to receive properties from within a widget.
+
+## Code example
+
+To pass properties down to a widget named `MyCustomWidget`, use the second argument of the `w` function:
+
+```ts
+import { w } from '@dojo/widget-core/d';
+
+w(MyCustomWidget, {
```

Comment:
nit: we use both `My` and `Your` as prefixes for example widgets, can we align to one of them?

Reply:
Done